tests/golden/helpers.bash

```diff
@@ -22,3 +22,6 @@ setup () {
 cleanOutput () {
   $@ | sed -r 's/\[[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}\]$/[2000-01-01 01:01:01]/g'
 }
+
+# Reset ANSI control sequences char.
+reset=$(printf '\x1b[0m')
```

Comment:
I think we now need tests specific for this use PR. We want to check that all commands that output a "field contents" append the `reset` sequence when necessary.

Reply:
Done